README.md

```diff
@@ -1,3 +1,26 @@
 # gradebook
 
-An R package to make course grade calculations easy, fast, and most importantly, correct. Course grades are produced from a data frame of assignment grades along with a grading policy file that specifies how they should be weighted when combined into a final grade.
\ No newline at end of file
+An R package to make course grade calculations easy, fast, and most importantly, correct. Course grades are produced from a data frame of assignment grades along with a grading policy file that specifies how they should be weighted when combined into a final grade.
+
+### Statement of Need
+
```

Comment:
It looks like this is a section in `paper/paper.md`. Does JOSS require this in both places?

Reply:
Yes, but it's not clear if it can be the same text in both places??